jcl/src/java.base/share/classes/com/ibm/tools/attach/target/AttachHandler.java

```diff
@@ -102,10 +110,20 @@
 	private static String nameProperty;
 	private static String pidProperty;
 	private static int numberOfTargets;
+	/*[IF Sidecar19-SE]*/
```

Comment:
Might be useful to add this code to Java 8 as well, but making the default true instead of false.

Reply:
Good idea. Done